firebase-perf/src/main/java/com/google/firebase/perf/transport/RateLimiter.java

```diff
@@ -63,7 +65,13 @@
    * @param capacity token bucket capacity
    */
   public RateLimiter(@NonNull Context appContext, final Rate rate, final long capacity) {
-    this(rate, capacity, new Clock(), getSamplingBucketId(), ConfigResolver.getInstance());
+    this(
+        rate,
+        capacity,
+        new Clock(),
+        getSamplingBucketId(),
```

Comment:
I think for `getSamplingBucketId()` we need to feed date time as the seed of the Random generator

Reply:
The `getSamplingBucketId` is a pure random number [Reference](https://github.com/firebase/firebase-android-sdk/blob/master/firebase-perf/src/main/java/com/google/firebase/perf/transport/RateLimiter.java#L72-L74). I don't think we need a specific seed.

Am I missing something here?

Reply:
I don't think that's needed. From java.util.random documentation:

> public Random()
Creates a new random number generator. This constructor sets the seed of the random number generator to a value very likely to be distinct from any other invocation of this constructor.

Reply:
Got it! Sounds good!